package-builder/extensions/apcu/build.sh

```diff
@@ -12,7 +12,7 @@ PNAME="gcp-php${SHORT_VERSION}-apcu"
 if [ ${SHORT_VERSION} == '56' ]; then
     download_from_pecl apcu 4.0.11
 else
-    download_from_pecl apcu
+    download_from_pecl apcu 5.1.20
```

Comment:
Why lock the version?

Reply:
How can I ensure that the latest version will be selected? Is it possible to specify a range of versions that can be used? (Similar to how composer or npm/yarn allows you to specify a minimum version, for example)